pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -220,12 +216,22 @@ func TestLocalTaskRef(t *testing.T) {
 		{
 			name:  "task-not-found",
 			tasks: []runtime.Object{},
-			ref: &v1beta1.TaskRef{
+			ref: &v1.TaskRef{
 				Name: "simple",
 			},
 			expected: nil,
 			wantErr:  true,
 		},
+		{
+			name:  "clustertask-not-found",
```

Comment:
is this a new test case? does it need to be added in this PR? it seems unrelated, but I may be missing something.

Reply:
I previously had the concern that when we were swapping to v1 while we don't have a v1.ClusterTaskRefKind would be an issue when we could not find a clusterTask so added this test case. But thanks to [Allow references to ClusterTasks in v1 Pipeline Tasks](https://github.com/tektoncd/pipeline/pull/6588) I think it's reasonable to not include this in the swap(opened https://github.com/tektoncd/pipeline/pull/6813).